### ISSUE TITLE
chrome43 but were moving in now stuck in chrome44. Solution Would you have any?

### ISSUE THREAD

**Issue by jin35rf** [jmpressjs/jmpress.js#179]

**Comment by FlorianBruniaux** [comment]
Hello, same problem here. 

No problem with Chrome **43**, but now, with the upgrade **44**, it does not work (no transitions ...)

Thank you in advance ;)

**Comment by Thoum** [comment]
Same problem here since i updated yesterday to chrome 44.

Thanks in advance !

**Comment by FlorianBruniaux** [comment]
Is there a way to show errors ? To understand where it blocks ...

**Comment by jin35rf** [comment]
Also stuck the individual site.

http://jmpressjs.github.io/jmpress.js/

**Comment by vontell** [comment]
Same issue here

**Comment by vontell** [comment]
Any leads on this problem?

**Comment by modiyash** [comment]
Jmpress just stopped working one fine day. Just checked that I do have Chrome 44 like everyon else here. Maybe that is the issue.

**Comment by deanshub** [comment]
Hi, I had the same problem, until @sokra will take a look my quick fix is changing the line
```
		return "-" + prefix + "-" + postfix;
```
in the jmpress.js file in the mapProperty function, to this line
```
		return (prefix!=="")?"-" + prefix + "-" + postfix:postfix;
```

**Comment by jin35rf** [comment]
deanshub > Thank you!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by tanmayrajani** [comment]
@deanshub Will that still work on older versions of chrome?

**Comment by deanshub** [comment]
yes it should, let me know if you have any problems with the fix and i'll try to help

**Comment by rendermouse** [comment]
@deanshub's fix solved my problem, too. Thanks!

**Comment by 12343954** [comment]
@deanshub ，Thank you so much ！！！！！！！！！！！！！！
